src/EnergyPlus/SZVAVModel.cc

```diff
@@ -237,7 +237,8 @@ namespace SZVAVModel {
                 }
             }
             FanCoilUnits::Calc4PipeFanCoil(state, SysIndex, SZVAVModel.ControlZoneNum, FirstHVACIteration, TempSensOutput, PartLoadRatio);
-            coilActive = state.dataLoopNodes->Node(coilAirInletNode).Temp - state.dataLoopNodes->Node(coilAirOutletNode).Temp;
+
+            coilActive = std::abs(state.dataLoopNodes->Node(coilAirInletNode).Temp - state.dataLoopNodes->Node(coilAirOutletNode).Temp) > 0;
```

Comment:
This is effectively what was happening (i.e. DT across the air inlet/outlet nodes means the coil is active 🤷) so, changing to make this explicit. Still smells wrong....

Reply:
This does seem correct when accounting for heating operation. I'm a little surprised because I've tested both cooling and heating for proper operation (i.e., graph of lines 171 to 184 above) and if this didn't work then the `if (!coilActive)` below would have turned off the water flow rate to the HW coil. Looking at your change I don't see how it could have worked correctly (i.e., -x > 0). Maybe I never tested this with a HW coil? I find that hard to believe but it's possible.